forge/ee/lib/billing/index.js

```diff
@@ -88,18 +96,20 @@ module.exports.init = function (app) {
             app.log.info(`Removing Project ${project.id} to Subscription for team ${team.hashid}`)
 
             if (projectItem) {
-                const metadata = projectItem.metadata ? projectItem.metadata : {}
-                metadata[project.id] = ''
+                const metadata = subscription.metadata ? subscription.metadata : {}
+                delete metadata[project.id]
                 const update = {
-                    quantity: projectItem.quantity - 1,
-                    metadata: metadata
+                    quantity: projectItem.quantity - 1
                 }
                 if (projectItem.quantity === 1) {
                     update.proration_behavior = 'always_invoice'
                 }
 
                 try {
-                    stripe.subscriptionItems.update(projectItem.id, update)
+                    await stripe.subscriptionItems.update(projectItem.id, update)
+                    await stripe.subscriptions.update(subscription.id, {
+                        metadata: metadata
+                    })
                 } catch (err) {
                     console.log(err)
```

Comment:
`console.log` should be swapped for proper app logging - and again, what is the consequence of this failing?

Reply:
Fixed to use `app.log.warn()`